analysis/webservice/algorithms/doms/config.py

```diff
@@ -14,39 +14,29 @@
 # limitations under the License.
 
 INSITU_API_ENDPOINT = 'https://doms.jpl.nasa.gov/insitu/1.0/query_data_doms_custom_pagination'
-INSITU_API_SCHEMA_ENDPOINT = 'https://doms.jpl.nasa.gov/insitu/1.0/cdms_schema'
+INSITU_API_SCHEMA_ENDPOINT = None
```

Comment:
Instead of None would it work if we pull the schema from https://doms.jpl.nasa.gov/in_situ_schema.json ?

Reply:
I'll try that out!